src/transformers/utils/import_utils.py

```diff
@@ -29,6 +29,7 @@
 from types import ModuleType
 from typing import Any, Tuple, Union
 
+import torch
```

Comment:
You can't import torch here, it has to be safely imported within the functions

Reply:
Thanks for this suggestion